apps/rush-lib/src/logic/deploy/Utils.ts

```diff
@@ -0,0 +1,16 @@
+function escapeRegExp(literal: string): string {
```

Comment:
I just put this all into a `Utils` file since I wasn't sure what the preference is for organizing these kinds of functions. Happy to move wherever you think is best.

Reply:
I thought about some ways to make this code more reusable, but it seems better to make it a glob.  I'll come back and deal with it later in a different PR that is all about globs.  It shouldn't hold up your deployment feature.